media/js/firefox/accounts.js

```diff
@@ -5,18 +5,9 @@
 (function(Mozilla, $) {
     'use strict';
 
-    var fxaEmail;
     var client = Mozilla.Client;
     var $main = $('main');
 
-    // check for fxaEmail in sessionStorage or URL params
-    try {
-        fxaEmail = sessionStorage.getItem('fxa-email');
```

Comment:
This seems to be from some past experiment, and I can find no reference to any page setting this any longer - so can only assume it's now fine to remove.

Reply:
FWIW, I think this was from an experiment on `/firerox/sync/`.